definitions/private-law/json/CaseEventToFields/SubmitAndPay/CaseEventToFields.json

```diff
@@ -71,34 +71,12 @@
     "PageDisplayOrder": 2,
     "FieldShowCondition": "payAgreeStatement=\"agree\""
   },
-  {
-    "LiveFrom": "02/12/2021",
```

Comment:
will this affect solicitor payment journey?

Reply:
Solicitor payment journey along with HWF is rebranded with this PR. So when we merge this to master code base, Solicitor payment and suppressed HWF change goes to Prod. But AwP remain suppressed in Prod. 

Solicitor payment has been tested along with AwP payment end to end with Libereta. not expecting any impact.